graf3d/eve7/inc/ROOT/REveElement.hxx

```diff
@@ -18,7 +18,11 @@
 
 #include <memory>
 
+#ifdef MISSING_JSON_FWD_HPP
```

Comment:
This may not work after ROOT is compiled when someone includes this header. The logic should probably be inverted (fallback to json.hpp should be default if nothing is defined), and `MISSING_JSON_FWD_HPP` should probably be `HAVE_JSON_FWD_HPP` and be generated in a header and used from there rather than added to the command line with `-D` as is being done now.